### DISCUSSION TITLE
FS-ATC - Cannot see radio is on

### DISCUSSION THREAD

**Discussion by Annolago** [flybywiresim/aircraft#5053] [Q&A] [answered]
Hi
Anyone here tried FS-ATC Chatter and have it working? My install works fine in X-plane 11 but in FS 2020 with FBW it can not see that the radio is turned on. It does register frequency changes perfectly, but not power state. 
Yes, I know, just use VATSIM instead - but don't always have time to file flight plan in advance..
Thanks...

**Comment by muddyfunkster** [comment, marked as answer]
there is an option in settings to power up radios with aircraft, disable it and it works.
